packages/themes/stories/UIKitThemeProvider.stories.tsx

```diff
@@ -26,26 +31,26 @@ const Template: Story = args => (
     appTheme={{
       colors: {
         // brand
-        brandPrimary: purple,
+        brandPrimary: themeBrandPrimary,
 
         // states
-        error: redLighten2,
+        error: themeError,
 
         // backgrounds
-        bgPrimary: greyDark,
-        bgPrimaryInverted: white,
-        bgDisabled: greyDarkLighten3,
-        bgHover: greyDarkLighten2,
+        bgPrimary: themeBgPrimary,
+        bgPrimaryInverted: themeBgPrimaryInverted,
+        bgDisabled: themeBgDisabled,
```

Comment:
I'm getting a TS error here, as well in `UIKitProvider.stories.tsx` where the `<UIKitProvider>` component is used. Are you able to see it as well or is it just something funky with my local setup? 

I was able to fix the TS errors by making all of the properties in the `LocalTheme` type optional, as well as casting the type of the CustomLinkComponent in `UIKitProvider.stories.tsx` like so:

`linkComponent={CustomLinkComponent as LinkComponent}`

But as I'm not too familiar with the ui-kit code base, I'm not 100% sure this won't break something.

Reply:
This has been a prior issue. I didn't want to make any component changes on this PR. I have that TODO to fix a few things related to the UIKitProvider. If you'd like to make those changes on another PR please feel free :) otherwise, I can.

Reply:
I've created a separate ticket for the TS errors, and I'm happy to pick it up: https://jira.d2iq.com/browse/D2IQ-82030